package.json

```diff
@@ -33,11 +41,11 @@
 		"test": "test"
 	},
 	"scripts": {
-		"default": "npm run build & npm run test & npm run docco",
-		"build": "npx rollup -c",
+		"default": "npm run lint && npm run build && npm run test && npm run docco",
+		"build": "npx rollup -c && cp package-cjs.json dist/cjs/package.json",
```

Comment:
there's a copy plugin for rollup if that's preferred?

Reply:
Why are we copying this? Why not just have it in `/dist/cjs/` in the first place?

Reply:
the `dist` folder isn't committed (and is typically deleted and rebuilt anyway). I tried to use `cat > ...` but it didn't like the `{}` (OS-specific I think)

Reply:
Fair enough. I guess it would be a hassle to restructure our `.gitignore` so that that particular file is preserved.